lib/config/validation.ts

```diff
@@ -118,6 +119,13 @@ export async function validateConfig(
   let errors: ValidationMessage[] = [];
   let warnings: ValidationMessage[] = [];
 
+  // Get env and argv configuration
+  const cfg = await parseConfigs(process.env, process.argv);
```

Comment:
I'm not sure why there's a need to parse the configs additionally due to this feature?